src/experimental/zookeeper/zookeeper/templates/cm.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "zookeeper.fullname" . }}
+  labels:
+    app: {{ template "zookeeper.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  zoo.cfg: |-
+    tickTime=2000
+    initLimit=10
+    syncLimit=5
+    dataDir=/data/zookeeper
+    clientPort=2181
+    # {{ .Values.replicaCount }}
+    {{ if eq ".Values.replicaCount)" "3" }}
```

Comment:
`{{ range .Values.replicaCount }}` can be used here to generate servers dynamically.